src/test/java/com/diffmin/AppTest.java

```diff
@@ -197,9 +206,22 @@ void should_apply_pure_move_patches(TestResources sources) throws Exception {
         runTests(sources);
     }
 
+    private static boolean isToIgnore(CtElement element, List<String> lines) {
+        if (element instanceof CtPackage
+                || element.isImplicit()
+                || element.getParent(CtElement::isImplicit) != null) {
```

Comment:
I had to do this because some children of implicit elements had `NoSourcePosition` attached to. This seemed like a good workaround because direct children of implicit elements should be implicit too.

If the default package-private constructor is implicit, its block will also be implicit because it comes attached to the constructor itself. I am unable to reason out precisely but it is very intuitive to me.